compiler/src/main/java/org/qbicc/graph/ConstructorElementHandle.java

```diff
@@ -0,0 +1,47 @@
+package org.qbicc.graph;
+
+import org.qbicc.type.ValueType;
+import org.qbicc.type.definition.element.ConstructorElement;
+import org.qbicc.type.definition.element.ExecutableElement;
+
+/**
+ * A handle for an instance constructor.
+ */
+public final class ConstructorElementHandle extends Executable {
```

Comment:
Shouldn't `ConstructorElementHandle` have a value dependency on `instance`?

Reply:
Oo, good catch, I thought I had found all of these.  When I add new nodes, missing the dependencies is generally the number one most common mistake that I make.  😕